browser/src/map/Clipboard.js

```diff
@@ -115,15 +119,15 @@ L.Clipboard = L.Class.extend({
 	_originWrapBody: function(body, isStub) {
 		var encodedOrigin = encodeURIComponent(this.getMetaURL());
 		var text =  '<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">\n' +
-		            '<html>\n' +
+		            '<div id="meta-origin" data-coolorigin="' + encodedOrigin + '"><html>\n' +
```

Comment:
It isn't actually valid html though, right, so could stop working if the sanitizing gets tightened up?

reminds me of the bodge for similar with gtk3 by adding an extra mimetype to identify that the clipboard content was coming from "ourself": https://cgit.freedesktop.org/libreoffice/core/commit/?id=88f7aae022bedd61588424a11bbc033217ba4e43

Reply:
Oh wow, I thought desktop is nice & simple, given we have a reliable way of getting notified of the ownership change & only browser is poor. Not. :-)

In any case, good point with the invalid HTML, I'll look at that. We had the markup in <head>, that is removed, so the first thought was to do this toplevel, which gives valid output currently, and I didn't tune this further. I'll check if it's possible to do this inside <body>, that would be very similar, but would meet both the sanitize and the validity requirements, I think.